lib/network/repository_notifications.dart

```diff
@@ -50,10 +50,10 @@ class ResponseNotificationItem {
   String get address => _address;
 
   ResponseNotificationItem(
-      this._Id, this._userId, this._timestamp, this._address);
+      this._id, this._userId, this._timestamp, this._address);
 
   ResponseNotificationItem.map(dynamic obj) {
-    _Id = obj["id"];
+    _id = obj["id"];
```

Comment:
the string "id" is different then "Id" from toMap - is this a bug?

Reply:
I think it's autogenerated from json. It's good that you renamed it.